pkg/minishift/constants/constants.go

```diff
@@ -22,6 +22,7 @@ const (
 	MinikubeIsoAlias       = "minikube"
 	OpenshiftContainerName = "origin"
 	OpenshiftExec          = "/usr/bin/openshift"
+	OpenshiftOcExec        = "/usr/bin/oc"
```

Comment:
Now that we have `oc binary` we should create a follow up issue about which places in the code it should use this `oc binary` and we should fix it.

Reply:
Not sure what you mean here. As I think all the interaction we have with OpenShift commander.